nativelib/src/main/resources/scala-native/stackOverflowGuards.c

```diff
@@ -252,13 +256,13 @@ static void setupSignalHandlerAltstack() {
         (size_t)alignToPageStart((void *)SIG_HANDLER_STACK_SIZE);
     handlerStack.ss_sp = malloc(handlerStack.ss_size);
     if (handlerStack.ss_sp == NULL) {
-        perror("Scala Native: StackOverflowGuards failed to allocate alternate "
+        perror("ScalaNative: StackOverflowGuards failed to allocate alternate "
```

Comment:
And the next 5 too?

Reply:
Fixed in https://github.com/scala-native/scala-native/pull/4310/commits/15cfff33b7cc030553c3c3eddbe4fad62d87de87.